TestShutterBlockBuilding: flaky

win: 
```
[WARN] [02-24|06:30:57.525] One ethash cache must always be in memory requested=0
--- FAIL: TestShutterBlockBuilding (3.57s)
    engine_api_tester.go:121: DBUG[02-24|06:30:55.054] [engine-api-tester] selected ports       sentry=46024 engineApi=46025 jsonRpc=46026
    node.go:319: INFO[02-24|06:30:55.079] Opening Database                         label=chaindata path=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestShutterBlockBuilding4219778523\\001\\chaindata
    kv_mdbx.go:331: INFO[02-24|06:30:55.140] [db] open                                label=chaindata sizeLimit=2TB pageSize=4KB
    kv_mdbx.go:333: DBUG[02-24|06:30:55.437] [db] open                                label=migrations sizeLimit=2TB pageSize=4KB
    node.go:385: INFO[02-24|06:30:55.758] Re-Opening DB in exclusive mode to apply migrations 
    kv_mdbx.go:331: INFO[02-24|06:30:55.842] [db] open                                label=chaindata sizeLimit=2TB pageSize=4KB
    migrations.go:235: INFO[02-24|06:30:55.843] Apply migration                          name=db_schema_version5
    migrations.go:280: INFO[02-24|06:30:56.006] Applied migration                        name=db_schema_version5
    migrations.go:235: INFO[02-24|06:30:56.006] Apply migration                          name=reset_stage_txn_lookup
    migrations.go:280: INFO[02-24|06:30:56.254] Applied migration                        name=reset_stage_txn_lookup
    migrations.go:285: INFO[02-24|06:30:56.349] Updated DB schema to                     version=7.0.0
    kv_mdbx.go:331: INFO[02-24|06:30:56.437] [db] open                                label=chaindata sizeLimit=2TB pageSize=4KB
    kv_mdbx.go:333: DBUG[02-24|06:30:56.662] [db] open                                label=temporary sizeLimit=1GB pageSize=4KB
    aggregator.go:157: DBUG[02-24|06:30:56.664] not generating new state-salt file as genNew=false 
    erigondb_settings.go:63: INFO[02-24|06:30:56.664] Creating erigondb.toml with default settings 
    erigondb_settings.go:65: INFO[02-24|06:30:56.664] erigondb settings                        step_size=1562500 steps_in_frozen_file=64
    aggregator.go:283: DBUG[02-24|06:30:56.736] history or ii disabled, can't register dependency domain=commitment
    aggregator.go:283: DBUG[02-24|06:30:56.736] history or ii disabled, can't register dependency domain=rcache
    genesis_write.go:155: INFO[02-24|06:30:56.781] Writing custom genesis block             hash=0x656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f
    config.go:888: INFO[02-24|06:30:57.180] Reading JWT secret                       path=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestShutterBlockBuilding4219778523\\001/jwt.hex
    config.go:908: INFO[02-24|06:30:57.180] Generated JWT secret                     path=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestShutterBlockBuilding4219778523\\001/jwt.hex
    node.go:319: INFO[02-24|06:30:57.181] Opening Database                         label=chaindata path=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestShutterBlockBuilding4219778523\\001\\chaindata
    kv_mdbx.go:331: INFO[02-24|06:30:57.216] [db] open                                label=chaindata sizeLimit=2TB pageSize=4KB
    kv_mdbx.go:333: DBUG[02-24|06:30:57.304] [db] open                                label=migrations sizeLimit=2TB pageSize=4KB
    backend.go:401: INFO[02-24|06:30:57.384] Initialised chain configuration          config="{ChainID: 1337, Terminal Total Difficulty: 0, Shapella: 1970-01-01 00:00:00 +0000 UTC, Dencun: 1970-01-01 00:00:00 +0000 UTC, Pectra: 1970-01-01 00:00:00 +0000 UTC, Fusaka: 1970-01-01 00:00:00 +0000 UTC, Glamsterdam: 1970-01-01 00:00:00 +0000 UTC, Engine: ethash}" genesis=0x656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f
    aggregator.go:161: INFO[02-24|06:30:57.480] generating new salt file 
    erigondb_settings.go:79: INFO[02-24|06:30:57.524] Reading DB settings from existing erigondb.toml 
    erigondb_settings.go:88: INFO[02-24|06:30:57.524] erigondb settings                        step_size=1562500 steps_in_frozen_file=64
    aggregator.go:283: DBUG[02-24|06:30:57.524] history or ii disabled, can't register dependency domain=commitment
    aggregator.go:283: DBUG[02-24|06:30:57.524] history or ii disabled, can't register dependency domain=rcache
    backend.go:1436: DBUG[02-24|06:30:57.525] [rpc] download of segments not complete yet. please wait StageSnapshots to finish 
    backend.go:595: INFO[02-24|06:30:57.525] Initialising Ethereum protocol           network=0
    kv_mdbx.go:333: DBUG[02-24|06:30:57.597] [db] open                                label=txpool sizeLimit=1TB pageSize=16KB
    filters.go:78: INFO[02-24|06:30:58.385] rpc filters: subscribing to Erigon events 
    ethbackend.go:260: DBUG[02-24|06:30:58.385] [rpc] new subscription to `newHeaders` events 
    txpool_grpc_server.go:268: INFO[02-24|06:30:58.386] New txns subscriber joined 
    aggregator.go:863: DBUG[02-24|06:30:58.386] [agg] merge                              collate_workers=1 merge_workers=1 compress_workers=1
    loop.go:94: WARN[02-24|06:30:58.386] Status error, sentry not ready yet       stream=PeerEvents err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    loop.go:94: WARN[02-24|06:30:58.386] Status error, sentry not ready yet       stream=RecvMessage err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    loop.go:94: WARN[02-24|06:30:58.386] Status error, sentry not ready yet       stream=RecvUploadMessage err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    loop.go:94: WARN[02-24|06:30:58.386] Status error, sentry not ready yet       stream=RecvUploadHeadersMessage err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    config.go:888: INFO[02-24|06:30:58.388] Reading JWT secret                       path=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestShutterBlockBuilding4219778523\\001/jwt.hex
    config.go:770: INFO[02-24|06:30:58.388] HTTP endpoint closed                     url=127.0.0.1:50522
    server.go:151: INFO[02-24|06:30:58.388] RPC server shutting down 
    backend.go:1206: EROR[02-24|06:30:58.388] cli.StartRpcServer error                 err="could not start RPC api: listen tcp 127.0.0.1:46026: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted."
    engine_server.go:163: EROR[02-24|06:30:58.388] [EngineServer] rpc server background goroutine failed err="could not start RPC api for engine: could not start RPC api: listen tcp 127.0.0.1:46025: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted."
    engine_server.go:165: DBUG[02-24|06:30:58.388] [EngineServer] engine rpc server goroutine terminated 
    peer_tracker.go:58: INFO[02-24|06:30:58.396] [p2p.peerTracker] running peer tracker component 
    mock_cl.go:147: DBUG[02-24|06:30:58.396] [mock-cl] building block                 timestamp=1771914655
    loop.go:94: WARN[02-24|06:30:58.396] Status error, sentry not ready yet       stream=PeerEvents-25413f7f-2eaa-4034-af83-f4ac885c7eb5 err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    message_listener.go:98: INFO[02-24|06:30:58.396] [p2p.message.listener] running p2p message listener component 
    pool.go:322: INFO[02-24|06:30:58.396] [txpool] Started 
    publisher.go:84: INFO[02-24|06:30:58.396] [p2p-publisher] running publisher component 
    loop.go:94: WARN[02-24|06:30:58.397] Status error, sentry not ready yet       stream=InboundMessages err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    loop.go:94: WARN[02-24|06:30:58.397] Status error, sentry not ready yet       stream=PeerEvents-Background err="no config for given genesis hash: unknown chain spec with genesis 656b170cf1b2335c23f187811392b33e98cc808177ba1c68b711713623f62f0f"
    block_listener.go:54: INFO[02-24|06:30:58.397] running block listener 
    engine_api_tester.go:243: 
        	Error Trace:	D:/a/erigon/erigon/execution/engineapi/engineapitester/engine_api_tester.go:243
        	            				D:/a/erigon/erigon/txnprovider/shutter/block_building_integration_test.go:230
        	            				D:/a/erigon/erigon/txnprovider/shutter/block_building_integration_test.go:63
        	Error:      	Received unexpected error:
        	            	build new payload failed: build new payload: fcu error: 403 Forbidden: signature is invalid
        	Test:       	TestShutterBlockBuilding
    block_listener.go:80: INFO[02-24|06:30:58.398] block listener stopped 
    backend.go:1522: INFO[02-24|06:30:58.398] [p2p] publisher goroutine terminated 
    backend.go:1505: INFO[02-24|06:30:58.398] [p2p] MessageListener goroutine terminated 
    backend.go:1514: INFO[02-24|06:30:58.398] [p2p] PeerTracker goroutine terminated 
    engine_server.go:140: DBUG[02-24|06:30:58.398] [EngineServer] engine log spammer goroutine terminated 
    backend.go:1216: EROR[02-24|06:30:58.398] [EngineServer] background goroutine failed err="could not start RPC api for engine: could not start RPC api: listen tcp 127.0.0.1:46025: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted."
    backend.go:1218: DBUG[02-24|06:30:58.398] [EngineServer] goroutine terminated 
    filters.go:315: DBUG[02-24|06:30:58.398] rpcdaemon: the subscription to pending blocks channel was closed 
    sync.go:507: DBUG[02-24|06:30:58.398] [1/6 OtterSync] error while executing stage err="OpenFolder: openFolder: context canceled"
    pool.go:2066: INFO[02-24|06:30:58.550] [txpool] stopped 
    backend.go:1591: INFO[02-24|06:30:58.550] [devp2p] txn pool goroutine terminated 
FAIL
coverage: 62.8% of statements
```

--- a/node/eth/backend.go
+++ b/node/eth/backend.go
@@ -1201,11 +1201,13 @@ func (s *Ethereum) Init(stack *node.Node, config *ethconfig.Config, chainConfig 
 		silkwormRPCDaemonService := silkworm.NewRpcDaemonService(s.silkworm, chainKv, settings)
 		s.silkwormRPCDaemonService = &silkwormRPCDaemonService
 	} else {
-		go func() {
-			if err := rpcdaemoncli.StartRpcServer(ctx, &httpRpcCfg, s.apiList, s.logger); err != nil {
+		s.bgComponentsEg.Go(func() error {
+			err := rpcdaemoncli.StartRpcServer(ctx, &httpRpcCfg, s.apiList, s.logger)
+			if err != nil && !errors.Is(err, context.Canceled) {
 				s.logger.Error("cli.StartRpcServer error", "err", err)
 			}
-		}()
+			return err
+		})
 	}
 
 	if chainConfig.Bor == nil || config.PolygonPosSingleSlotFinality {

